FND/Law/LegalCore.rdf

```diff
@@ -108,7 +109,7 @@
 		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://dictionary.reference.com/browse/constrains</fibo-fnd-utl-av:definitionOrigin>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-fnd-law-cor;hasInforce">
+	<owl:ObjectProperty rdf:about="&fibo-fnd-law-cor;hasInForce">
```

Comment:
Why no domain of Jurisdiction? Note that FIBO has no "Situation" concept.
The definition uses "currently in force" which is a) circular and b) implies that there should be a period (possibly past or future) associated with the relationship - which makes sense to me for FIBO e.g. certain trades might be controlled from a certain date.

Reply:
Happily, in order to address BE-44 and incorporate a more complete lattice, as we've discussed, I'm adding the concept of a situation, including a restriction on holdsDuring.  Given that passes muster, we could then leverage the concept of a situation here.

Reply:
That should be done in a separate issue, as you've noted.